playbooks/sample-monitoring-sap.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name:  sap monitoring configuration
+  hosts: 127.0.0.1
+  connection: local
+  gather_facts: no
+  tasks:
+
+    - name: Execute the  role monitoring_sap
+      # include_role:
+      import_role:
+        name: { role: ../roles/monitoring_sap }
+      when: sap_monitoring_action == "add"
+
+      ##  Delete a SAP System from monitoring
+    - name: Check existence of the variables defined in roles/monitoring_sap/vars/main.yml
+      fail:
+        msg: 'variable {{ sap_monitoring_nr }} not defined in the file roles/monitoring_sap/vars/main.yml'
+      when: sap_monitoring_nr is not defined
+
+    - name: include role
```

Comment:
All these should be moved to main.yml. Action in main.yml decides what tasks must be executed

Reply:
done